src/App.tsx

```diff
@@ -1,26 +1,44 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from './app/hooks';
+import { setTodos } from './features/todos';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [isListLoading, setIsListLoading] = useState(false);
+  const dispatch = useDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    setIsListLoading(true);
+    getTodos()
+      .then(array => {
+        dispatch(setTodos(array));
+      })
+      .finally(() => setIsListLoading(false));
+  }, [dispatch]);
```

Comment:
The dependency array for useEffect is correct if you only want to run this effect once when the component mounts. However, if 'getTodos' or 'setTodos' can change or if you want to refetch todos when certain dependencies change, you might need to include those dependencies in the array.